src/routes/business/businesses.ts

```diff
@@ -0,0 +1,217 @@
+import _ from "lodash";
+import moment from "moment";
+import uuid from "node-uuid";
+
+import {
+  saveBusiness,
+  redlock,
+  setBusinessOrigin,
+  getBusiness,
+  findBusinesses,
+} from "../../db";
+
+import generateID from "../../helpers/id";
+import { storeBusinessInSortedSet } from "../../helpers/businesses";
+import { BusinessWebhookEvent, MockBusiness } from "../../helpers/types";
+import { triggerWebhook } from "../../helpers/webhooks";
+
+export const createBusiness = async (req, res) => {
+  const businessId = generateID(); // Do not exceed 36 characters
+  let createdBusiness;
+  const businessLockKey = `redlock:${process.env.MOCKSOLARIS_REDIS_PREFIX}:business:${businessId}`;
+
+  await redlock.using([businessLockKey], 5000, async (signal) => {
+    if (signal.aborted) {
+      throw signal.error;
+    }
+    const createdAt = moment();
+
+    const business = {
+      ...req.body,
+      id: businessId,
+      createdAt: createdAt.toISOString(),
+    };
+
+    createdBusiness = await saveBusiness(business).then(() => {
+      res.status(201).send({
+        id: businessId,
+        ...req.body,
+      });
+    });
+
+    await storeBusinessInSortedSet(business);
+
+    if (req.headers.origin) {
+      await setBusinessOrigin(businessId, req.headers.origin);
+    }
+  });
+
+  return createdBusiness;
+};
+
+export const showBusiness = async (req, res) => {
+  const { business_id: businessId } = req.params;
+  try {
+    const business = await getBusiness(businessId);
```

Comment:
we need to map stored business to the response. Later this stored redis object will include other business properties like beneficial owner, legal representative , which we do should not include into the response